packages/cli/src/lib/webpack/prerender.js

```diff
@@ -37,7 +37,7 @@ module.exports = function (env, params) {
 			throw err;
 		}
 
-		handlePrerenderError(err, env, stack, entry);
+		await handlePrerenderError(err, env, stack, entry);
```

Comment:
Prerender errors result in a good ol' `process.exit(1)`, so it's better if we wait than have Webpack continue on even after we know the build is doomed.